planter/planter.sh

```diff
@@ -69,7 +69,7 @@ RUN_OPTS="${RUN_OPTS} -it"
 RUN_OPTS="${RUN_OPTS} --hostname=planter"
 
 # - NOTE: SELinux disabled for this container to prevent relabeling $HOME (!)
-RUN_OPTS="${RUN_OPTS} --security-opt label:disable"
+RUN_OPTS="${RUN_OPTS} --security-opt label=disable"
```

Comment:
this matches https://docs.docker.com/engine/reference/run/#security-configuration